src/Monitor.c

```diff
@@ -30,6 +32,22 @@ static struct Monitor_InterfaceActivationEntry *activation_log_buffer = NULL;
 static struct Monitor_InterfaceActivationEntry activation_log_buffer[RT_EXEC_LOG_SIZE];
 #endif
 
+#define STACK_BYTE_PATTERN 0xA5
+
+// Where the pattern goes in the stack area is dependent upon
+// whether the stack grow to the high or low area of the memory.
+#if (CPU_STACK_GROWS_UP == TRUE)
+
+  #define CALCAULATE_USED_STACK(_low, _size, _high_water) \
```

Comment:
Is there a reason why this needs to be a macro instead of a function?

Reply:
Replaced with inline function